### DISCUSSION TITLE
how to resume job from savepoint use java api code?

### DISCUSSION THREAD

**Discussion by sanguoasd** [apache/flink-cdc#1433] [Q&A]
I create cdc job by using java api code(env.execute(jobName)), it can be cancel by restapi, but can't find any api to resume the job. Flink official examples show use flink command(flink run -s  jar) to resume job, anybody know how to use java code to resume the job? thank you very much.

**Comment by sanguoasd** [comment]
Finally, I find a way to pass config to flink env like :

```
        Configuration configuration = new Configuration();
        configuration.setString(SavepointConfigOptions.SAVEPOINT_PATH, "//savepointPath");
        configuration.setBoolean(SavepointConfigOptions.SAVEPOINT_IGNORE_UNCLAIMED_STATE, true);
        StreamExecutionEnvironment env = StreamExecutionEnvironment.createRemoteEnvironment(
                flinkProperties.getIp(),
                flinkProperties.getPort(), configuration);
```

**Comment by gtk96** [comment]
please refer:
https://ververica.github.io/flink-cdc-connectors/master/content/connectors/mysql-cdc%28ZH%29.html#id17
